manifests/init.pp

```diff
@@ -6,7 +6,7 @@
 
   # JVM Settings
   $javahome                                                      = undef,
-  Enum['openjdk-11', 'oracle-jdk-1.8', 'custom'] $jvm_type       = 'openjdk-11',
+  Enum['openjdk-11', 'custom'] $jvm_type       = 'openjdk-11',
```

Comment:
$version still defaults to 5.7.1. Is 5.7.1 still supported by atlassian? Does it support running on jdk11?

Reply:
I've looked into this now.
According to https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html 7.19 is the oldest supported version with EOL 13 Dec 2024 
Since 7.x is still supported on java8 and Oracle continue to support java8 for paying customers https://www.oracle.com/java/technologies/java-se-support-roadmap.html it's a bit to early to remove support in the module.